### PR TITLE
Fix row width for full-width char

### LINKED ISSUE #39: beautifultable is not beautiful for multibyte string
Hi, 
I'm trying to print multibyte string in beautifultable. But the result is not beautiful, unsuitable number of space.

like this:
```python
from beautifultable import BeautifulTable
table = BeautifulTable()
table.column_headers = ['Language', 'Hello']
table.append_row(['Japanese', 'こんにちは'])
table.append_row(['English', 'Hello'])
table.append_row(['Chinese', '你好'])
table.append_row(['Emoji', '😉'])
print(table)
```
```
+----------+-------+
| Language | Hello |
+----------+-------+
| Japanese | こんにちは |
+----------+-------+
| English  | Hello |
+----------+-------+
| Chinese  |  你好   |
+----------+-------+
|  Emoji   |   😉   |
+----------+-------+
```

Is there any way to solve this problem?
Thanks.

```
$ pip freeze | grep beautifultable
beautifultable==0.5.3
```

### PR DESCRIPTION
**Pull request by showwin**
I fixed #39.

the results changed:
screen shot 2018-11-30 at 18 34 19

### CODE REVIEW

File: beautifultable/utils.py [hunk 1]
```diff
@@ -55,7 +60,9 @@ def get_output_str(item, detect_numerics, precision, sign_value):
 
 def ansilen(item):
```
Comment (showwin) [inline beautifultable/utils.py]: This method is no longer `ansilen()`, now is `east_asian_len()`.
But I could not come up with good name. 
Is there any better idea for this method.

Comment (pri22296) [reply beautifultable/utils.py]: Don't remove the existing contents of the ansilen method. Keep the name and content intact as it has been used in many other places which could break something. Just add on top of it. Does this also work for european based unicode characters?

Comment (showwin) [reply beautifultable/utils.py]: >Does this also work for european based unicode characters?
It doesn't work... I'll fix it.

result:
screen shot 2018-12-03 at 23 04 23

Comment (showwin) [reply beautifultable/utils.py]: I fixed this problem: 36733b5553949f55168d4ca368aa8fbd430fbe71
screen shot 2018-12-03 at 23 38 56


### Detail
European based unicode characters are treated `'A'` (Ambiguous) by `east_asian_width()`. So I used `len(ANSI_REGEX.sub('', char))` only for ambiguous character.
like this: (this is the capture before fixing) 
screen shot 2018-12-03 at 23 21 59

Comment (showwin) [reply beautifultable/utils.py]: Do you have something to worry about? If you have, please give me concrete examples. I may fix it 👍

Comment (showwin) [reply beautifultable/utils.py]: >Maybe add some comments explaining why you're doing some things

👍 

>can you just test this once with some long strings ensuring that this doesn't break when rows are wrapped.

OMG...
screen shot 2018-12-05 at 23 34 53

I'll try to fix.

File: test.py [hunk 2]
```diff
@@ -384,5 +385,15 @@ def test_table_auto_width(self):
 
         self.assertEqual(len_for_max_width_80, len_for_max_width_200)
 
+
+class UtilsTestCase(unittest.TestCase):
+    def test_ansilen(self):
+        self.assertEqual(ansilen('Hello'), 5)  # English
+        self.assertEqual(ansilen('こんにちは'), 10)  # Japanese
+        self.assertEqual(ansilen('你好'), 4)  # Chinese
+        self.assertEqual(ansilen('😉'), 2)  # Emoji
```
Comment (pri22296) [inline test.py]: This seems to be returning 1 on python version != 3.6. You might wanna check that.
https://travis-ci.org/pri22296/beautifultable/jobs/462833273

File: beautifultable/utils.py [hunk 3]
```diff
@@ -55,7 +61,14 @@ def get_output_str(item, detect_numerics, precision, sign_value):
...
     """Returns the length of the string after stripping ansi escape sequences"""
-    return len(ANSI_REGEX.sub('', item))
+    total_width = 0
+    for char in item:
+        char_type = east_asian_width(char)
```
Comment (pri22296) [inline beautifultable/utils.py]: Getting this error on Python version 2.7
https://travis-ci.org/pri22296/beautifultable/jobs/462833270
`east_asian_width() argument 1 must be unicode, not str`

**Comment by pri22296** [comment]
I'm closing this PR because the functionality has been added in 5de5af4

**Pull request closed without merging**